docs/marketplace/user/features/202108.0/marketplace-merchant-portal-product-offer-management-feature-overview.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Marketplace Merchant Portal Product Offer Management feature overview
+description: This document describes product offer management in the Merchant Portal.
+template: concept-topic-template
+---
+
+In a marketplace environment, merchants manage [product offers](/docs/marketplace/user/features/{{ page.version }}/marketplace-product-offer-feature-overview.html) in the Merchant Portal.
+
+
+## Managing product offers
+
+Merchants create product offers based on the products that exist in the marketplace. When creating an offer, they can see full information about products, including the number of existing offers for each of them.
+
+One merchant can create multiple offers for the same product.
+
+When creating an offer, they can define the following settings:
+
+|SETTING|DESCRIPTION|
+|---|---|
+| Status: Active or Inactive |
+| Merchnat SKU | Unique identifier of the merchants' offer. |
```

Comment:
Unique identifier of the merchants' offer is Offer Reference, the Merchant SKU is when the Merchant is using a different SKU in his/her system than the one used in the marketplace for this product